### DISCUSSION TITLE
The fault in time domain simulation

### DISCUSSION THREAD

**Discussion by Tang115** [CURENT/andes#235] [General]
Dr. Cui

I like to know is there any way to adjust the phase-to-phase impedance in a fault? I only see fault-to-ground impedance in your document.

Also, is there any reference to this fault model/equation?

Thanks!

Best

**Comment by cuihantao** [comment]
There is no fault type that models phase-to-phase impedance. The existing `Fault` is a three-phase-to-ground type fault. 

See Milano's Power System Modelling and Scripting or PSAT's manual.
